hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/web3j/TestWeb3jService.java

```diff
@@ -366,13 +366,13 @@ private void contractPersist(String binary, long entityId) {
                 .persist();
     }
 
-    private void historicalContractPersist(String binary, long entityId, final Address contractAddress) {
+    private void historicalContractPersist(String binary, EntityId entityId, final Address contractAddress) {
         final var contractBytes = Hex.decode(binary.replace(HEX_PREFIX, ""));
         final var entity = domainBuilder
                 .entity()
```

Comment:
Same comment in any other class about setting realm and shard via helper.
```suggestion
                .entity(entityId)
```

Reply:
Fixed